src/main/webapp/WEB-INF/views/admin/license/licenseBulkReg-js.jsp

```diff
@@ -0,0 +1,246 @@
+            data : jsonData,
+            colNames:['id', 'License Name','License Type','Notice','Source Code','SPDX Short Identifier','Nickname',
+                'Website for the license','User Guide',  'License Text', 'Attribution','Comment', 'Status'],
+            colModel: [
+                { name: 'id', 	index: 'id', width: 75, key:true, hidden: true, editable:false},
+                { name: 'licenseName', index: 'License Name', width: 200, align: 'left', editable:false},
+                { name: 'licenseType', index: 'License Type', width: 200, align: 'left', editable:false},
+                { name: 'obligationNotificationYn', index: 'Notice', width: 75, align: 'left', editable:false},
+                { name: 'obligationDisclosingSrcYn', index: 'Source Code', width: 75, align: 'left', editable:false},
+                { name: 'shortIdentifier', index: 'SPDX Short Identifier', width: 300, align: 'left', editable:false},
+                { name: 'licenseNicknames', index: 'Nickname', width: 300, align: 'left', editable:false},
+                { name: 'webpages', index: 'Website for the license', width: 200, align: 'left', editable:false},
+                { name: 'description', index:'User Guide', width: 250, align: 'left', editable:false},
+                { name: 'licenseText', index:'License Text', width: 150, align: 'left', editable:false},
+                { name: 'attribution', index:'Attribution', width: 150, align: 'left', editable:false},
+                { name: 'comment', index:'comment', width: 150, align: 'left', editable:false},
+                { name: 'status', index:'status', width: 150, align: 'left'}
+            ],
+            viewrecords: true,
+            rowNum: ${ct:getConstDef("DISP_PAGENATION_DEFAULT")},
+            rowList: [${ct:getConstDef("DISP_PAGENATION_LIST_STR")}],
+            autowidth: true,
+            gridview: true,
+            height: 'auto',
+            pager: '#pager',
+
+            autoencode: true,
+            editurl:'clientArray',
+            recordpos:'right',
+            toppager:true,
+            loadonce:false,
+            cellsubmit : 'clientArray',
+            ignoreCase: true,
+            // multiselect: true,
+
+            loadComplete: function(data) {
+                _mainLastsel = -1;
+            },
+            // ondblClickRow: function(rowid,iRow,iCol,e) {
+            //     if(rowid) {
+            //         if (rowid != _mainLastsel) {
+            //             $("#list").jqGrid('saveRow', _mainLastsel);
+            //         }
+            //
+            //         editable()
+            //
+            //         $("#list").jqGrid('editRow', rowid);
+            //         _mainLastsel = rowid;
+            //
+            //         var nextCol = $("#list").jqGrid('getGridParam', 'colModel')[iCol].name
+            //         var nextRow = rowid
+            //         $('#' + nextRow + "_" + nextCol).focus();
+            // }
```

Comment:
If you don't need a function, it's good to delete it.

Reply:
thank you for good review